rasa_core/training/interactive.py

```diff
@@ -1062,7 +1062,7 @@ def _enter_user_message(sender_id: Text,
                         endpoint: EndpointConfig) -> None:
     """Request a new message from the user."""
 
-    question = questionary.text("Next user input (Ctr-c to abort):")
+    question = questionary.text("Your input ->")
```

Comment:
I think we should keep a message about how to exit (we can print that separately though)

Reply:
where would you print that? we can also keep the `ctr-c` thing and also add it to the scaffold at the cli

Reply:
I'd print that right at the beginning but maybe it is not to important and users just now how this works, @JustinaPetr thoughts on this?

Reply:
@JustinaPetr

Reply:
Sorry for the late response. I would say we should keep it printed, because it's not really clear how to exit. Ctr-c is an obvious choice by habit, but to have a good UX, I would suggest having it printed. RIght after beginning looks like a great place for that.

Reply:
Okay cool! that's already done :-)